share/CMakeLists.txt

```diff
@@ -20,6 +20,13 @@ CONFIGURE_FILE(
     COPYONLY
     )
 
+EXECUTE_PROCESS(COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/dbtypes_gen.py ${core_version} OUTPUT_VARIABLE DBTYPES_JSON_ENTRIES)
```

Comment:
You should be able to do something like:

```cmake
EXECUTE_PROCESS(COMMAND python {CMAKE_CURRENT_SOURCE_DIR}/dbtypes_gen.py ${core_version} 
  OUTPUT_VARIABLE DBTYPES_JSON_ENTRIES 
  RESULT_VARIABLE res_var)
if(NOT "${res_var}" STREQUAL "0")
  message(FATAL_ERROR "process failed res_var='${res_var}'")
endif()
```

Note that I am splitting up onto multiple lines for readability.